pages/CheckingPageTable.js

```diff
@@ -4,17 +4,19 @@ class CheckingPageTable extends BasePage {
     
     get newCheckingConfirmation () {return $('#new-account-conf-alert')}
 
-    get switchChecking () { return $('.switch-label')}
+    get switchChecking () { return $('//*[contains(text(),"Individual Checking")]//ancestor::div[@class="card-body"]//span[@class="switch-handle"]')}
 
-    get checkingBody () { return $('.even')}
+    get switchChecked () { return $('//*[contains(text(),"Individual Checking")]//ancestor::div[@class="card-body"]//input[@class="switch-input"]')}
 
     get checkingTransactionsTableNextButton () {return $('#transactionTable_next')}
 
     get checkingTransactionsTablePreviousButton () {return $('#transactionTable_previous')}
 
     get checkingTableTextInfo(){return $('#transactionTable_info')}
 
-    get checkingTableFilter(){return $('tbody')}
+    get getCellFromForRow(){return $(`#transactionTable>tbody>tr:nth-child(${row})>td:nth-child(2)`)}
+
+    get getTableNumberOfRows(){return $('tbody')}
```

Comment:
This is returning a web element representing the whole table. It's not returning the number of rows the table has. Use a selector that matches the rows of the table, and use array based syntax to get them into an array of row elements. Then just return its length, like:

```javascript
let rows = await $$("tbody > tr");
return rows.length
```

This is not a getter method, its a regular async method. Implement it without the `get` modifier